test-compiler-plugins/ballerina-plugin/src/main/java/io/ballerina/examplestaticcodeanalyzer/CustomRule.java

```diff
@@ -0,0 +1,45 @@
+package io.ballerina.examplestaticcodeanalyzer;
```

Comment:
Usually we have multiple levels for package name like this `io.ballerina.runtime.observability.metrics`
better to rename this